src/dsp_tools/xmllib/value_converters.py

```diff
@@ -48,13 +47,15 @@ def replace_newlines_with_tags(text: str, converter_option: NewlineReplacement)
 
 def replace_newlines_with_paragraph_tags(text: str) -> str:
     """
-    Replace `Start\\nEnd` with `<p>Start</p><p>End</p>`
+    Replaces one newline `Start\\nEnd` with `<p>Start</p><p>End</p>`
+
+    Replaces multiple newlines `Start\\n\\nEnd` with `<p>Start</p><p>End</p>`
```

Comment:
For me, this is ambiguous. I think this would be clearer:

```suggestion
    Replace `Start\\nEnd` with `<p>Start</p><p>End</p>`
    Multiple consecutive newlines will be treated as one newline:
    `Start\\nMidde\\n\\nEnd` becomes `<p>Start</p><p>Middle</p><p>End</p>`
```